resources/js/components/Elements/Slider.vue

```diff
@@ -72,13 +88,13 @@ export default {
             if (!this.stopOnHover) {
                 return
             }
-            this.hover = useElementHover(this.slider)
+            this.hover = useElementHover(this.$el)
         },
 
         scroll() {
             this.position = this.vertical ? this.slider.scrollTop : this.slider.scrollLeft
-            this.showLeft = this.position
-            this.showRight = this.slider.offsetWidth + this.position < this.slider.scrollWidth - 1
+            this.showLeft = this.loop ? this.loop : this.position
+            this.showRight = this.loop ? this.loop : this.slider.offsetWidth + this.position < this.slider.scrollWidth - 1
```

Comment:
```suggestion
            this.showLeft = this.loop || this.position
            this.showRight = this.loop || this.slider.offsetWidth + this.position < this.slider.scrollWidth - 1
```
This should do the same, but test it 😇